protobuf/definitions/service.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package definitions;
```

Comment:
```suggestion
package definition;
```

Why the plural on the package name? @ilgooz suggested a singular and I also prefer the singular.